curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b4d5446c81404d2d092cf.md

```diff
@@ -0,0 +1,109 @@
+---
+id: 646b4d5446c81404d2d092cf
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+# --description--
+
+You will create a second `option` HTML element with the `value` attribute set to `yyyy-mm-dd` and set the Text inside to `Year, Month, Day`
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed